pkg/app/app.go

```diff
@@ -34,6 +38,14 @@ type AgentSettings struct {
 	ExitFileLocation string `json:"exit_file_location"`
 }
 
+type TelemetrySettings struct {
+	Enabled bool `json:"enabled"`
+	// The telemetry agent reads the defined number of lines from syslog file of the system and pushes to the server
+	SysLogNumberOfLinesToRead int `json:"syslog_number_of_lines_to_read"`
+	// At what frequency to push the telemetry data to the server
+	PushFrequency string `json:"push_frequency"`
```

Comment:
Add a description of the unit that is accepted here as well.

Reply:
Added